Neos.Neos/Classes/Fusion/Cache/NeosFusionContextSerializer.php

```diff
@@ -65,30 +64,29 @@ public function normalize(mixed $object, string $format = null, array $context =
     }
 
     /**
-     * @param array<int|string,mixed> $serializedNode
+     * @param array<string,mixed> $serializedNode
      */
     private function tryDeserializeNode(array $serializedNode): ?Node
     {
-        $contentRepositoryId = ContentRepositoryId::fromString($serializedNode['contentRepositoryId']);
-
-        $contentRepository = $this->contentRepositoryRegistry->get($contentRepositoryId);
-
-        $workspace = $contentRepository->getWorkspaceFinder()->findOneByName(WorkspaceName::fromString($serializedNode['workspaceName']));
-        if (!$workspace) {
+        $nodeAddress = NodeAddress::fromArray($serializedNode);
+
+        $contentRepository = $this->contentRepositoryRegistry->get($nodeAddress->contentRepositoryId);
+
+        try {
+            $subgraph = $contentRepository->getContentGraph($nodeAddress->workspaceName)->getSubgraph(
+                $nodeAddress->dimensionSpacePoint,
+                $nodeAddress->workspaceName->isLive()
```

Comment:
OH NO, we shouldn't repeat the errors of our past, this can of condition is not and never was a good idea IMHO.

Reply:
This is obviously the same as it was before, so nothing this change needs to fix, but as I was seeing it, I really do not like it.

Reply:
yeah me neither but im not sure yet how to solve it ... i introduced it myself in Neos 9 :D